packages/react-composites/src/composites/CallComposite/adapter/createHandlers.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { CallAgent } from '@azure/communication-calling';
+import { CallingHandlers, createDefaultCallingHandlers } from '@internal/calling-component-bindings';
+/* @conditional-compile-remove(teams-identity-support)) */
+import { createDefaultTeamsCallingHandlers, TeamsCallingHandlers } from '@internal/calling-component-bindings';
+import {
+  CallCommon,
+  StatefulCallClient,
+  StatefulDeviceManager,
+  TeamsCallAgent,
+  _isACSCall,
+  _isACSCallAgent,
+  _isTeamsCall,
+  _isTeamsCallAgent
+} from '@internal/calling-stateful-client';
+
+/**
+ * @private
+ */
+export type CallHandlersOf<AgentType extends CallAgent | TeamsCallAgent> = AgentType extends CallAgent
+  ? CallingHandlers
```

Comment:
Do you need this conditional compilation? Or can you set it up such that for stable build `TeamsCallingHandlers` is `never`?

Reply:
This is the only place to call TeamsCallingHandlers outside of their package, so I guess keeping it here would be nice to avoid adding another type into stable api as a never type (we will have to export TeamsCallingHandlers in communication-react)